azure-pipelines-20230430.yml

```diff
@@ -201,35 +201,30 @@ stages:
         displayName: UT FT other modules
         timeoutInMinutes: '240'
         steps:
-          - task: Maven@4
-            displayName: maven install
+          - task: Docker@2
+            displayName: "build docker image"
             inputs:
-              mavenPomFile: 'pom.xml'
-              goals: 'clean install'
-              options: $(MVN_OPTS_INSTALL)
-              publishJUnitResults: true
-              testResultsFiles: '**/surefire-reports/TEST-*.xml'
-              jdkVersionOption: '1.8'
-          - task: Maven@4
-            displayName: UT other modules
+              containerRegistry: 'onehouse-docker-hub'
```

Comment:
lets keep onehouse out of this PR. Can we just use the apachehudi docker hub

Reply:
Yeah, it is just a name for the connection. I can replace it.